README.md

```diff
@@ -24,7 +24,11 @@ screensaver off.
 https://extensions.gnome.org/extension/517/caffeine/
 
 ## Installation from git
-
+```
+make build
+make install
+```
+Alternatively:
```

Comment:
I think we can remove the manual commands below and rely on the Makefile only

Reply:
Done :)

Reply:
Thanks!

Feel free to open an other PR if you'd like to improve the README more